x-pack/filebeat/input/cel/input.go

```diff
@@ -1250,7 +1250,7 @@ func cloneMap(dst, src mapstr.M) {
 // walkMap walks to all ends of the provided path in m and applies fn to the
 // final element of each walk. Nested arrays are not handled.
 func walkMap(m mapstr.M, path string, fn func(parent mapstr.M, key string)) {
-	//nolint:typecheck // typecheck linter is buggy and thinks rest is not used.
+	//nolint:typecheck,nolintlint // typecheck linter is buggy and thinks rest is not used.
```

Comment:
Please remove this.

Reply:
I disagree. This been used before in this repo in the other place.
The linter in CI flags this and this is false positive, please look at the code and the linter error above.
This is also not a new code and not related to my change.
The linter on my local machine doesn't fail here as well, so I can't reproduce the issue.

Reply:
The linter directive here is being used to work around a bug. This is described in the issue I linked above.